rollbar/invites.go

```diff
@@ -6,27 +6,31 @@ import (
 	"net/http"
 )
 
+type ListInvitesResultResponse struct {
```

Comment:
Actually, I think I would rather call it `ListInvitesResponseResult` because its the "result of the response".